Makefile

```diff
@@ -113,6 +113,11 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+.PHONY: render-manifest
+render-manifest: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
Help text with "Deploy" said me that this yaml will be deployed. IMO this target only create yaml without deploying. Could we change this description?

Reply:
fixed